.github/workflows/docker-images.yaml

```diff
@@ -7,9 +7,19 @@ on:
   # On each commit merged into main, push sha and branch patterns to prefect-dev
   push:
     branches: main
+    paths:
+      - 'Dockerfile'
```

Comment:
Changing `.dockerignore` could also alter the contents of the image, worth adding that one

Reply:
Oh we also need the `./ui` path: https://github.com/PrefectHQ/prefect/blob/main/Dockerfile#L31

Reply:
A few more that rarely change but that could technically influence how the package is built:

* MANIFEST.in
* setup.cfg
* versioneer.py
* .gitignore
* .gitattributes

Reply:
great calls! updated in [cc7989c](https://github.com/PrefectHQ/prefect/pull/12163/commits/cc7989c408fc0ae1fc744a845e8fdc6f6b952f27)